src/runtime_src/core/edge/drm/zocl/zocl_ctrl_ert.c

```diff
@@ -957,6 +956,33 @@ static void zert_cmd_query_cu(struct zocl_ctrl_ert *zert, struct xgq_cmd_sq_hdr
 	}
 }
 
+static void zert_cmd_query_mem(struct zocl_ctrl_ert *zert, struct xgq_cmd_sq_hdr *cmd,
+			       struct xgq_com_queue_entry *resp)
+{
+	struct xgq_cmd_query_mem *c = (struct xgq_cmd_query_mem *)cmd;
+	struct xgq_cmd_resp_query_mem *r = (struct xgq_cmd_resp_query_mem *)resp;
+	struct drm_zocl_dev *zdev = zocl_get_zdev();
+
+	init_resp(resp, cmd->cid, 0);
+
+	switch (c->type) {
+        case XGQ_CMD_QUERY_MEM_ADDR:
+		r->l_mem_info = (u32)zdev->host_mem;
+		r->h_mem_info = (u32)((zdev->host_mem & 0xFFFFFFFF00000000LL) >> 32);
+		break;
+
+	case XGQ_CMD_QUERY_MEM_SIZE:
+		r->l_mem_info = (u32)zdev->host_mem_len;
+		r->h_mem_info = (u32)((zdev->host_mem_len & 0xFFFFFFFF00000000LL) >> 32);
```

Comment:
Value "0xFFFFFFFF00000000LL" shows up twice, right? Can we define it as a macro?